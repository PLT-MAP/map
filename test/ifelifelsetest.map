func main(){
	if(x<5){
		cost=5;
	}
	elif (x<10)
	{
		cost=10;
	}
	else{
		cost=15;
	}
}
	"""
