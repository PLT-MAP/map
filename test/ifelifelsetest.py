import networkx as nx
import sys
def main():
    x = 2.0
    if (x<5.0 and x>10.0):
        print("success")
    elif (x<10.0 or x>20.0):
        cost = 10.0
    else:
        cost = 15.0
    print(str("cost = ") + str(cost))
if __name__ == '__main__': 
	try:
		main()
	except:
		print'Error:',sys.exc_info()[0]
		print'Resolve error before running again!'