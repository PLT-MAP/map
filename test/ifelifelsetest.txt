def main():
	if(x<5.0):
		cost=5.0
	elif(x<10.0):
		cost=10.0
	else:
		cost=15.0

