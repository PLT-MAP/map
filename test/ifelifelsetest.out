import networkx as nx
import sys
def main():
    x = 2.0
    if (x<5.0):
        print("success")
    elif (x<10.0):
        cost = 10.0
    else:
        cost = 15.0
if __name__ == '__main__': 
	try:
		main()
	except:
		print'Error:',sys.exc_info()[0]
		print'Resolve error before running again!'